docs/components/concepts/encryption-at-rest.md

```diff
@@ -0,0 +1,72 @@
+- You can only select the encryption type when [creating a cluster](/docs/components/console/manage-clusters/create-cluster.md). You cannot change the encryption type after cluster creation.
+- You can configure encryption keys on a per-cluster basis so that each cluster has a dedicated encryption key. Encryption keys can be configured for all cluster versions.
+- You can view cluster encryption key details in **Cluster details** on the **Console Overview** tab.
+
+:::note
+Backups use the default provider GCP encryption.
+:::
+
+### Encryption types
+
+The following table summarizes the available types of cluster encryption at rest.
+
+| Encryption type                   | Managed by | Protection level                                                                                                                                                                                                                                                                                                                                                                                   |
+| :-------------------------------- | :--------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Provider encryption key (default) | Google     | Google uses a [FIPS 140-2](https://cloud.google.com/security/compliance/fips-140-2-validated) validated encryption module (certificate 4407) in our production environment.                                                                                                                                                                                                                        |
+| Software encryption key           | Camunda    | <p><ul><li><p>Google KMS [software](https://cloud.google.com/docs/security/key-management-deep-dive#software_backend_software_protection_level) protection level.</p></li><li>Cryptographic operations are performed in software.</li><li>Compliant with [FIPS 140-2 Level 1](https://cloud.google.com/docs/security/key-management-deep-dive#fips_140-2_validation).</li></ul></p>                |
+| Hardware encryption key           | Camunda    | <p><ul><li><p>Google KMS [hardware](https://cloud.google.com/docs/security/key-management-deep-dive#backend_hardware_protection_level) protection level.</p></li><li>Cryptographic operations are performed in a hardware security module (HSM).</li><li>Compliant with [FIPS 140-2 Level 3](https://cloud.google.com/docs/security/key-management-deep-dive#fips_140-2_validation).</li></ul></p> |
+
+## Provider encryption key (default)
+
+By default, Camunda 8 SaaS cluster data at rest is protected using Google Cloud Platform (GCP) encryption.
```

Comment:
I think since you fully spelled out GCP above, we can keep this as just "GCP encryption".

Reply:
Thanks - great spot, edited 👍